scipy/spatial/transform/tests/test_rotation_groups.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import division, print_function, absolute_import
+
+import pytest
+
+import numpy as np
+from scipy.spatial.transform import Rotation
+from scipy.optimize import linear_sum_assignment
+from scipy.spatial.distance import cdist
+from scipy.constants import golden as phi
+from scipy.spatial import cKDTree
+
+
+TOL = 1E-12
+NS = range(1, 13)
+NAMES = ["I", "O", "T"] + ["C%d" % n for n in NS] + ["D%d" % n for n in NS]
+SIZES = [60, 24, 12] + list(NS) + [2 * n for n in NS]
+
+
+def _register(P, Q):
```

Comment:
I don't understand the meaning of this name. Why `register`?

Reply:
I was thinking in terms of "registering points": https://en.wikipedia.org/wiki/Point_set_registration

I have changed it to calculate the root-mean-square distance instead.  I think this is better.  The function is now called `_calculate_rmsd`.